docs/install/install-cave-beta.md

```diff
@@ -0,0 +1,165 @@
+# Install CAVE - BETA Version
+
+CAVE is the **C**ommon **A**WIPS **V**isualization **E**nvironment that is used for rendering and analyzing data for AWIPS.  The installer may require administrator priviledges to install and may require other system changes (environment variables, etc) as well.
+
+---
+
+## Latest CAVE Versions
+
+- [**Linux: 20.3.2-0.1**](#linux)
+- [**Windows:**](#windows) VM option available, but direct install not yet supported
+- **Mac:** not yet supported
+
+[**View release notes**](https://www.unidata.ucar.edu/blogs/news/tags/awips-release)
+
+!!! note "Version 20.* of CAVE is not compatible with Version 18.* EDEX and vice versa, Version 18.* of CAVE is not compatible with Version 20.* EDEX."
+
+---
+
+## General Requirements
+
+Regardless of what Operating System CAVE is running on, these general requirements are recommended in order for CAVE to perform optimally:
+
+- Local machine
+
+    !!! error "Running CAVE via X11 forwarding or ssh tunneling is **not** supported. Using a [VNC connection is the only remote option](../appendix/common-problems.md#remotely-connecting-to-cave), and may result in worse performance than running locally."
+  
+- Java 11
```

Comment:
I don't know if it's worth mentioning Python3?

Reply:
I was wondering about actually removing Java11 as well since it's not something the user has to install since it's installed with the rpms